### ISSUE TITLE
Lists with class open do not work

### ISSUE THREAD

**Issue by mitya57** [rtfd/sphinx_rtd_theme#354]
reStructuredText has a distinction between “compact” and “open” lists. When a list has an `open` class, all its items are wrapped in `<p>` tags so that they have more spacing. For example, this block of reStructuredText:
```rest
.. rst-class:: open

* Item one
* Item two
* Item three
```

Produces the following output:
```html
<ul class="open">
<li><p class="first">Item one</p>
</li>
<li><p class="first">Item two</p>
</li>
<li><p class="first">Item three</p>
</li>
</ul>
```

Compared to compact lists:
```html
<ul class="compact simple">
<li>Item one</li>
<li>Item two</li>
<li>Item three</li>
</ul>
```

However with the RTD theme all lists render as compact, because the CSS sets `margin-bottom: 0` for `p:last-child`:

![Screenshot with the RTD theme](https://cloud.githubusercontent.com/assets/1381584/22066537/c4c2d072-dda6-11e6-9a08-fc40c5a8aeb7.png)

For comparison, this is how it looks with the default theme (Alabaster):

![Screenshot with Alabaster theme](https://cloud.githubusercontent.com/assets/1381584/22066566/e77f677e-dda6-11e6-8819-9fcaccc0e22e.png)

Support for “open” lists would be nice to have as compact lists look ugly when the list items contain multiple paragraphs or images.

**Comment by Blendify** [comment]
I am confused, is there documentation that this is supposed to work?
I tried looking in the Sphinx docs, restructured text docs, alabaster, and the w3 school docs and don't see any. Can you provide information about this?

**Comment by mitya57** [comment]
It is briefly mentioned in Docutils documentation: [one](http://docutils.sourceforge.net/docs/user/config.html#compact-lists) and [two](http://docutils.sourceforge.net/docs/user/latex.html#lists). Note that in Sphinx the `class` directive [becomes `rst-class`](http://www.sphinx-doc.org/en/stable/rest.html#id3).

Our problem was that we often use multiple paragraphs in list items, and the spacing between items is less than the spacing between paragraphs in one item, which makes it difficult to visually distinguish the list items. Here is a screenshot:

> ![rtd-theme-lists](https://user-images.githubusercontent.com/1381584/34174146-08207422-e509-11e7-96e0-d96c4078c5f5.png)

We would like the spacing between lists be at least same or even larger. Using the `open` class would potentially fix this, but unfortunately that does not help.

**Comment by Blendify** [comment]
Thanks I should have time to take a look this week

**Comment by Blendify** [comment]
This should be fixed with bb82cd22fd84600348be221f407918656fd6aa19 if not please open a new report.

**Comment by wbolster** [comment]
@Blendify   thanks a lot for addressing this :1st_place_medal:!
